src/librustc_data_structures/sync.rs

```diff
@@ -497,13 +497,15 @@ impl<T> Once<T> {
     /// If the value was already initialized the closure is not called and `false` is returned,
     /// otherwise if the value from the closure initializes the inner value, `true` is returned
     #[inline]
```

Comment:
Shouldn't the comment be changed to reflect the change that instead of returning a `bool` we now return a `&T`?

Reply:
Definitely :), going to do that now.